packages/cli-repl/src/cli-repl.ts

```diff
@@ -174,7 +182,7 @@ class CliRepl {
    */
   greet(): void {
     console.log(`Using MongoDB: ${this.mdbVersion} \n`);
-    console.log(TELEMETRY);
+    if (!this.disableGreetingMessage) console.log(TELEMETRY);
```

Comment:
@mmarcon so I was thinking the telemetry greeting was getting kind of annoying. So I am thinking of hiding once a user either runs `enableTelemetry` or `disableTelemetry`. What do you think? Or would you always prefer it there?

Reply:
How hard would it be to show it only the first time I launch the shell? Essentially, I'd do something like this (tweaking the copy a bit):

**1st time**
```
$ mongosh

Connecting to: mongodb://127.0.0.1:27017
Using MongoDB: 4.2.0

To help improve our products, anonymous usage data is collected and sent to MongoDB periodically (https://www.mongodb.com/legal/privacy-policy).
You can opt-out by running the disableTelemetry() command.

$ mongosh >
```

**Next time**
```
$ mongosh

Connecting to: mongodb://127.0.0.1:27017
Using MongoDB: 4.2.0

$ mongosh >
```
@fredtruman @Sgrinfy happy to get some feedback on the copy.

Reply:
not hard at all. if that's what we want, it's easily manageable.

Reply:
@lrlna Yes, that's what I would do. If it's not something you can get done quickly before today's internal release no worries. You can merge this PR and we can ticket this.

Reply:
ok, fixed! i have another PR incoming with actual segment implementation that hopefully you can have with today's release as well.